src/dfmt/formatter.d

```diff
@@ -1551,6 +1558,12 @@ private:
                 immutable l = indents.indentToMostRecent(tok!"switch");
                 if (l != -1 && config.dfmt_align_switch_statements == OptionalBoolean.t)
                     indentLevel = l;
+                else if (astInformation.structInfoSortedByEndLocation
+                    .canFind!(st => st.startLocation < current.index && current.index < st.endLocation)) {
+                    immutable l2 = indents.indentToMostRecent(tok!"{");
+                    assert(l2 != -1);
```

Comment:
are you sure this l2 can't be -1? I would put in a reason why it should work as assert message (second string argument) so that in case this ever breaks it is possible to debug it and for others reading it to know why it is used without understanding all the code context.